<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TournamentGridWidget</class>
 <widget class="QWidget" name="TournamentGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>483</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить как Excel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>245</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Изменить размер клетки</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
